src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -350,6 +351,7 @@
   "InteractiveImport": "Interactive Import",
   "InteractiveSearch": "Interactive Search",
   "Interval": "Interval",
+  "IsAvailable": "Considered Available",
```

Comment:
Let's change the key for this as well to make it clear "IsAvailable" to "ConsideredAvailable"

Reply:
No problem - will commit changes in a few minutes.